app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageActionsBox.kt

```diff
@@ -31,33 +31,40 @@ import com.wire.android.util.debug.LocalFeatureVisibilityFlags
 @Composable
 fun MessageComposeActionsBox(
     transition: Transition<MessageComposeInputState>,
-    messageComposerState: MessageComposerInnerState,
-    isMentionActive: Boolean
+    messageComposeInputState: MessageComposeInputState,
+    isMentionActive: Boolean,
+    startMention: () -> Unit,
+    onAdditionalOptionButtonClicked: () -> Unit,
+    modifier: Modifier = Modifier,
 ) {
-    Column(
-        Modifier
-            .wrapContentSize()
-    ) {
+    Column(modifier.wrapContentSize()) {
         Divider()
         Box(Modifier.wrapContentSize()) {
             transition.AnimatedContent(
-                contentKey = { state -> state != MessageComposeInputState.Enabled },
+                contentKey = { state -> state is MessageComposeInputState.Active },
                 transitionSpec = {
                     slideInVertically { fullHeight -> fullHeight / 2 } + fadeIn() with
                             slideOutVertically { fullHeight -> fullHeight / 2 } + fadeOut()
                 }
             ) { state ->
-                if (state != MessageComposeInputState.Enabled)
-                    MessageComposeActions(messageComposerState, isMentionActive)
+                if (state is MessageComposeInputState.Active)
+                    MessageComposeActions(
+                        messageComposeInputState.attachmentOptionsDisplayed,
```

Comment:
perhaps we can pass the boolean flag instead of the whole MessageComposeInputState ?

Reply:
right, we can take that from the transaction,  no need to pass `messageComposeInputState` as a parameter